examples/fungible/README.md

```diff
@@ -0,0 +1,270 @@
+below.
+
+First setup a local network with two wallets, and keep it running in a separate terminal:
+
+```bash
+./scripts/run_local.sh
+```
+
+Compile the `fungible` application WebAssemlby binaries, and publish them as an application
+bytecode:
+
+```bash
+export LINERA_WALLET="$(realpath target/debug/wallet.json)"
+export LINERA_STORAGE="rocksdb:$(dirname "$LINERA_WALLET")/linera.db"
+export LINERA_WALLET_2="$(realpath target/debug/wallet_2.json)"
+export LINERA_STORAGE_2="rocksdb:$(dirname "$LINERA_WALLET_2")/linera_2.db"
+
+cd examples/fungible && cargo build --release && cd ..
+linera --wallet "$LINERA_WALLET" --storage "$LINERA_STORAGE" publish-bytecode \
+examples/target/wasm32-unknown-unknown/release/fungible_{contract,service}.wasm
+```
+
+This will output the new bytecode ID, e.g.:
+
+```rust
+e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65010000000000000001000000
+```
+
+## Creating a Token
+
+In order to use the published bytecode to create a token application, the initial state must be
+specified. This initial state is where the tokens are minted. After the token is created, no
+additional tokens can be minted and added to the application. The initial state is a JSON string
+that specifies the accounts that start with tokens.
+
+In order to select the accounts to have initial tokens, the command below can be used to list the
+chains created for the test:
+
+```bash
+linera --storage "$LINERA_STORAGE" --wallet "$LINERA_WALLET" wallet show
+```
+
+A table will be shown with the chains registered in the wallet and their meta-data. The default
+chain should be highlighted in green. Each chain has an `Owner` field, and that is what is used for
+the account.
+
+The example below creates a token application where two accounts start with the minted tokens, one
+with 100 of them and another with 200 of them:
+
+```bash
+linera --storage "$LINERA_STORAGE" --wallet "$LINERA_WALLET" create-application $BYTECODE_ID \
+    --json-argument '{ "accounts": {
+        "User:445991f46ae490fe207e60c95d0ed95bf4e7ed9c270d4fd4fa555587c2604fe1": "100.",
+        "User:c2f98d76c332bf809d7f91671eb76e5839c02d5896209881368da5838d85c83f": "200."
+    } }'
+```
+
+This will output the application ID for the newly created token, e.g.:
+
+```rust
+e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65010000000000000001000000e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65030000000000000000000000
+```
+
+## Using the Token Application
+
+Before using the token, a source and target address should be selected. The source address should
+ideally be on the default chain (used to create the token) and one of the accounts chosen for the
+initial state, because it will already have some initial tokens to send. The target address should
+be from a separate wallet due to current technical limitations. To see the available chains in the
```

Comment:
(Really? But that's just a limitation of the frontend, is it?)

Reply:
I should have elaborated on this, but `linera service` locks the wallet file but only runs in the scope of a single chain from that wallet :/

Reply:
Ah, right, makes sense.